app/common/services/strataService.js

```diff
@@ -197,6 +197,25 @@ angular.module('RedhatAccess.common').factory('strataService', [
                     return deferred.promise;
                 }
             },
+            articles: {
+                get: function (uri) {
+                    var deferred = $q.defer();
+                    var splitUri = uri.split('/');
+                    uri = splitUri[splitUri.length - 1];
```

Comment:
I literally copy and pasted from the service part fetching solutions.  We could definitely rewrite it to be more sane but I want going to spend too much time on it in hopes that we can get this functionality in uds.  I have a Jira open for getting the HTML in solutions and articles.  Feel free to reject this though and I'll rewrite it on Monday.